### PR TITLE
Convert military time to regular time

### PR DESCRIPTION
**Pull request by deebarizo**
Using the variables hourInt and minuteInt, this converts military time to regular time.

### CODE REVIEW

File: lambda-tools/src/components/Randomizer/Randomizer.js [hunk 1]
```diff
@@ -50,11 +50,11 @@ function Randomizer() {
   const convertMilitaryTimeToRegularTime = (hourInt, minuteInt) => {
     if (hourInt >= 13) { return `${hourInt - 12}:${minuteInt.toString().padStart(2, '0')} PM`; }
 
-    if (hourInt === 12) { return `${hourInt}:${minuteInt.toString().padStart(2, '0')} PM`; }
+    if (hourInt == 12) { return `${hourInt}:${minuteInt.toString().padStart(2, '0')} PM`; }
```
Comment (michaelagard) [inline lambda-tools/src/components/Randomizer/Randomizer.js]: `==` should be `===` in React.

File: lambda-tools/src/components/Randomizer/Randomizer.js [hunk 2]
```diff
@@ -50,11 +50,11 @@ function Randomizer() {
...
 
-    if (hourInt === 0) { return `${hourInt}:${minuteInt.toString().padStart(2, '0')} AM`; }
+    if (hourInt == 0) { return `12:${minuteInt.toString().padStart(2, '0')} AM`; }
```
Comment (michaelagard) [inline lambda-tools/src/components/Randomizer/Randomizer.js]: `==` should be `===` in React.

File: lambda-tools/src/components/Randomizer/Randomizer.js (hunk 1 above)
Comment (deebarizo) [reply lambda-tools/src/components/Randomizer/Randomizer.js]: Thanks, I fixed it.

File: lambda-tools/src/components/Randomizer/Randomizer.js (hunk 2 above)
Comment (deebarizo) [reply lambda-tools/src/components/Randomizer/Randomizer.js]: Thanks, I fixed it.

**Comment by michaelagard** [comment]
I've changed the logic of the Randomizer component. I don't want your code to go to waste though, could you repull and make the changes needed to integrate into the new logic if it isn't that big of a change?

**Comment by deebarizo** [comment]
I can do it but I'm still getting a bug. See here: https://i.ibb.co/3k1FcC2/Screen-Shot-2019-09-10-at-8-11-35-PM.png

Can you fix that first? Thanks.

**Pull request closed without merging**